package.json

```diff
@@ -76,7 +76,6 @@
     "gulp-sourcemaps": "~2.6.1",
     "gulp-util": "~3.0.8",
     "jest": "^22.0.3",
-    "mustache": "^2.2.1",
```

Comment:
I'm not sure what this was doing here. Removing it isn't related to the rest of the changes, but as far as I can see this dependency isn't used anywhere (anymore?).

Reply:
Well, mustache was used to generate the .mst template, right?
So it’s in fact related to the other changes you did :)

Reply:
Well true :) What I meant was that the `mustache` used for this generator was declared in `client/package.json` – so there shouldn't have been one for that purpose in the root package.json.